src/main/webapp/app/components/login/LoginPage.tsx

```diff
@@ -66,6 +63,33 @@ export default class LoginPage extends React.Component<ILoginProps> {
     }
   }
 
+  @computed
+  get trialActivationLink() {
+    if (this.props.authenticationStore.loginError) {
+      const errorMessage = getErrorMessage(
+        this.props.authenticationStore.loginError
+      );
+      if (errorMessage.endsWith(UNAUTHORIZED_LICENSE_AGREEMENT_NOT_ACCEPTED)) {
+        const trialActivationKey = this.props.authenticationStore.loginError
+          .response?.body.trialActivationKey;
+        if (trialActivationKey) {
+          const to = `/account/active-trial/finish?key=${trialActivationKey}`;
```

Comment:
Could you use ACCOUNT_ACTIVE_TRIAL_FINISH? We probably should create a TrialActivationPageLink in UrlUtils.tsx.